### DISCUSSION TITLE
Can we use hybrid mode with db-less?

### DISCUSSION THREAD

**Discussion by secat** [Kong/kong#8722] [Help] [answered]
Is it possible to use Kong using the hybrid mode combined with the db-less feature?

If no, why is not supported?

**Comment by hbagdi** [comment, marked as answer]
Hybrid mode has two types of Kong nodes:
- Control-plane (CP) nodes
- Data-plane (DP) nodes

DP nodes are DB-less.
Are you asking if you can run CP nodes without a database?
If so, what is the problem that you are trying to solve here?
